components/activity_log_modal/activity_log_modal.test.jsx

```diff
@@ -4,9 +4,10 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 
+import ActivityLogModal from 'components/activity_log_modal/activity_log_modal';
+
 import {General} from 'mattermost-redux/constants';
 
-import ActivityLogModal from 'components/activity_log_modal/activity_log_modal.jsx';
```

Comment:
Can we convert the test file to TS as well?

Reply:
I think we still need to convert this test file to Typescript :)

Reply:
@majidsajadi We're still looking to have this test file converted to TypeScript. Let us know if you need any help with that.